### PR TITLE
added a new speedUp function and increased the version

### PR DESCRIPTION
**Pull request by sparsh459**

### CODE REVIEW

File: vevestaX/vevesta.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def __init__(self):
         self.__variables = {}
         self.__filename = self.get_filename()
         self.__sampleSize = 0
+        self.__skipEDA = False
```
Comment (Priyanka-Vevesta) [inline vevestaX/vevesta.py:30]: Name is self.speedUp = False

**Review by Priyanka-Vevesta — CHANGES_REQUESTED** [review]

File: vevestaX/vevesta.py (hunk 1 above)
Comment (sparsh459) [reply vevestaX/vevesta.py:30]: the function have same name as variable so name should not be self.skillUp

File: vevestaX/vevesta.py [hunk 2]
```diff
@@ -53,7 +54,8 @@ def dataSourcing(self, value):
             self.__dataSourcing = value.columns
             self.__data=value
             self.__sampleSize=len(value)
-            self.__correlation = value.corr(method='pearson')
+            if self.speedUp == True:
```
Comment (Priyanka-Vevesta) [inline vevestaX/vevesta.py:57]: if you are speeding up why will this be true ? it will be false.

File: vevestaX/vevesta.py [hunk 3]
```diff
@@ -83,7 +85,8 @@ def featureEngineering(self, value):
                 self.__featureEngineering = cols
 
         if type(value) == pandas.core.frame.DataFrame:
-            self.__correlation = value.corr(method='pearson')
+            if self.speedUp == True:
```
Comment (Priyanka-Vevesta) [inline vevestaX/vevesta.py:88]: same reason

File: vevestaX/vevesta.py [hunk 4]
```diff
@@ -317,14 +321,16 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
 
             df_messages.to_excel(writer, sheet_name='messages', index=False)
             pandas.DataFrame(sampledData).to_excel(writer,sheet_name='sampledata',index=False)
+            
+            if self.speedUp == True:
```
Comment (Priyanka-Vevesta) [inline vevestaX/vevesta.py:324]: this will be false

File: vevestaX/vevesta.py [hunk 5]
```diff
@@ -317,14 +321,16 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
...
+                    to_excel(writer, sheet_name='EDA-correlation', index=True)
 
-            if self.__correlation is not None:
-                pandas.DataFrame(self.__correlation).style.\
-                applymap(self.__colorCellExcel).\
-                applymap(self.__textColor).\
-                to_excel(writer, sheet_name='EDA-correlation', index=True)
-
-        self.__missingEDAValues(filename)
+        if self.speedUp == True:
```
Comment (Priyanka-Vevesta) [inline vevestaX/vevesta.py:331]: this will be false

File: vevestaX/vevesta.py [hunk 6]
```diff
@@ -317,14 +321,16 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
...
+                    applymap(self.__textColor).\
+                    to_excel(writer, sheet_name='EDA-correlation', index=True)
 
-            if self.__correlation is not None:
-                pandas.DataFrame(self.__correlation).style.\
-                applymap(self.__colorCellExcel).\
-                applymap(self.__textColor).\
-                to_excel(writer, sheet_name='EDA-correlation', index=True)
-
-        self.__missingEDAValues(filename)
```
Comment (Priyanka-Vevesta) [inline vevestaX/vevesta.py:327]: this will be __EDA(filename)

**Review by Priyanka-Vevesta — CHANGES_REQUESTED** [review]

**Review by Priyanka-Vevesta — APPROVED** [review]

**Pull request merged**